packages/smooth_app/lib/background/background_task_details.dart

```diff
@@ -8,6 +9,28 @@ import 'package:smooth_app/data_models/operation_type.dart';
 import 'package:smooth_app/database/local_database.dart';
 import 'package:smooth_app/query/product_query.dart';
 
+/// Stamps we can put on [BackgroundTaskDetails].
+///
+/// With that stamp, we can de-duplicate similar tasks.
+enum BackgroundTaskDetailsStamp {
+  basicDetails('BD'),
```

Comment:
can we have something more explicit, and language dependant ?
basic_details_fr

Reply:
It's not meant to be read by humans, but we can make something more explicit like `basic_details`.

The language wouldn't be put in that `enum` anyway; here we just say "btw it's basic details" to a task that already knows the language, the country, the barcode and the changes.
The _task_ stamp is currently built this way: `'$barcode;detail;$stamp'` (yeah I know, I forged "stamp" because I was fed up with "key" and "id", but here we have the _task_ stamp and the _details_ stamp, probably confusing).

Anyway, the task stamp is here to say which tasks are "similar".
Why do we need that? In order to keep the order of the user inputs related to the same stamp, which frees us from waiting for ALL the previous tasks for being successful even if they not related.

If a data has no language variants (e.g. nutrition facts) its stamp should not have a language component.